deploy/00_deploy.ts

```diff
@@ -40,7 +42,7 @@ module.exports = async (hre: any) => {
       log: true,
     });
 
-    console.log(address, f4);
+    console.log(`Contract address: ${address}`);
```

Comment:
It was a bit confuding to log the f4 address for the actor who deployed the contract here, since it doesn't belong to the contract eth address